src/lib/utils/referrerUtils.ts

```diff
@@ -1,5 +1,17 @@
 import logger from "../Logger";
 
+/**
+ *
```

Comment:
It would be good to add here some general information what this functions is for.